packages/utils/src/configured_bignumber.ts

```diff
@@ -34,4 +34,7 @@ if (isNode) {
     };
 }
 
+const orig = BigNumber.config;
+BigNumber.config = (_args?: any) => orig({});
```

Comment:
Should probably document why we do this and also clobber `BigNumber.set()`.

Also, maybe this is neater?
```ts
(orig => BigNumber.config = (...args: any) => orig({}))(BigNumber.config);
BigNumber.set = BigNumber.config;
```